### PR TITLE
Add Sunil_Tajpuriya to Before I Die

### PR DESCRIPTION
**Pull request by tajpuriya27**
I am adding `Sunil Tajpuriya` to Before I Die.

### CODE REVIEW

File: CONTRIBUTION-GUIDELINES.md
Comment (XanderRubio) [inline CONTRIBUTION-GUIDELINES.md:1]: Hi @tajpuriya27,

Thank you for your time in assisting and contributing! Can you please confirm if you intend to make changes to the CONTRIUBTION-GUIDELINES.md file as when reviewing it is showing that you have made changes, and I wanted to make sure you intended to do so and then ask what did you do differently before I merge. Please let me know and thank you!

**Review by XanderRubio — CHANGES_REQUESTED** [review]
Excellent your text and image show up great on the preview deployment, and striving to live a purposeful life of rishes and supporting and providing for your parents is noble, and I wish you the best in your pursuit Additionally, congratulations on being the first contributor from Nepal! It's a unique flag. Please confirm if you intend to make changes to the CONTRIBUTION-GUIDELINES.md. After, I will go ahead and merge.

File: CONTRIBUTION-GUIDELINES.md
Comment (tajpuriya27) [reply CONTRIBUTION-GUIDELINES.md:1]: There are no specific changes on the CONTRIBUTION-GUIDELINES.md file. The extension in my IDE has put extra line spaces between lines in the md file. 
I think you should ignore the changes on that file. Thank you for your quick response!

Comment (XanderRubio) [reply CONTRIBUTION-GUIDELINES.md:1]: Great, and thank you, @tajpuriya27, for the clarification!

**Review by XanderRubio — APPROVED** [review]
Excellent @tajpuriya27! Your contribution looks great on the website, and I am going ahead with merging your work now. Thank you for your time in contributing and sharing that you want to make your parents proud. Make it happen!

**Comment by XanderRubio** [comment]
@tajpuriya27   Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark. Make it happen for living a life of value and making your parents proud!

 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Sunil! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day, and I will be sending you an invite to be a member of the Before I Die Code now.

**Pull request merged**